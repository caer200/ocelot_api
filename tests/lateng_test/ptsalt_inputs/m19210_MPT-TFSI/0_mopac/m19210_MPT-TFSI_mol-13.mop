CHARGES

 
C	1.454069 +1 5.025480 +1 1.785085 +1
S	1.894743 +1 3.282771 +1 2.099573 +1
F	2.269308 +1 5.525242 +1 0.862716 +1
F	1.582837 +1 5.761867 +1 2.888954 +1
F	0.203853 +1 5.143792 +1 1.352436 +1
N	0.736873 +1 2.821889 +1 3.065002 +1
O	1.688819 +1 2.666256 +1 0.828185 +1
O	3.222462 +1 3.333238 +1 2.636848 +1
S	0.911210 +1 2.503270 +1 4.594984 +1
C	1.432666 +1 0.756440 +1 4.591912 +1
O	1.972245 +1 3.203117 +1 5.259411 +1
O	-0.383540 +1 2.484443 +1 5.201381 +1
F	2.613838 +1 0.614510 +1 3.994377 +1
F	1.555046 +1 0.309706 +1 5.832939 +1
F	0.550454 +1 0.009358 +1 3.953180 +1
