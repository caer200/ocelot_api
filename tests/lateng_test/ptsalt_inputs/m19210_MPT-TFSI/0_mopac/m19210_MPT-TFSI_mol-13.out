 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:10:05 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          1.45406900  *   5.02548000  *   1.78508500  *
     2       S          1.89474300  *   3.28277100  *   2.09957300  *
     3       F          2.26930800  *   5.52524200  *   0.86271600  *
     4       F          1.58283700  *   5.76186700  *   2.88895400  *
     5       F          0.20385300  *   5.14379200  *   1.35243600  *
     6       N          0.73687300  *   2.82188900  *   3.06500200  *
     7       O          1.68881900  *   2.66625600  *   0.82818500  *
     8       O          3.22246200  *   3.33323800  *   2.63684800  *
     9       S          0.91121000  *   2.50327000  *   4.59498400  *
    10       C          1.43266600  *   0.75644000  *   4.59191200  *
    11       O          1.97224500  *   3.20311700  *   5.25941100  *
    12       O         -0.38354000  *   2.48444300  *   5.20138100  *
    13       F          2.61383800  *   0.61451000  *   3.99437700  *
    14       F          1.55504600  *   0.30970600  *   5.83293900  *
    15       F          0.55045400  *   0.00935800  *   3.95318000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          1.4541      5.0255      1.7851
     2         S          1.8947      3.2828      2.0996
     3         F          2.2693      5.5252      0.8627
     4         F          1.5828      5.7619      2.8890
     5         F          0.2039      5.1438      1.3524
     6         N          0.7369      2.8219      3.0650
     7         O          1.6888      2.6663      0.8282
     8         O          3.2225      3.3332      2.6368
     9         S          0.9112      2.5033      4.5950
    10         C          1.4327      0.7564      4.5919
    11         O          1.9722      3.2031      5.2594
    12         O         -0.3835      2.4844      5.2014
    13         F          2.6138      0.6145      3.9944
    14         F          1.5550      0.3097      5.8329
    15         F          0.5505      0.0094      3.9532

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C2 N O4 S2 F6  =    15 atoms



      MOLECULAR POINT GROUP   :   C1  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1          C      2      3      4      5
      2          S      7      8      6      1
      3          F      1
      4          F      1
      5          F      1
      6          N      2      9
      7          O      2
      8          O      2
      9          S     11     12      6     10
     10          C      9     13     14     15
     11          O      9
     12          O      9
     13          F     10
     14          F     10
     15          F     10

     Ion Atom No.  Type    Charge
 
    1       6      N       +1

          COMPUTED CHARGE ON SYSTEM: +1

          COMPUTED CHARGE ON SYSTEM: +1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
